docker/Makefile

```diff
@@ -21,6 +21,18 @@ service-base:
 	python3 ../ci/jinja2_render.py '{"base_image":{"image":"base"}}' Dockerfile.service-base Dockerfile.service-base.out
 	docker build -t service-base -f Dockerfile.service-base.out --cache-from service-base,$(SERVICE_BASE_LATEST),base ..
 
+.PHONY: hail-public-image
+hail-public-image:
+	make -C ../hail wheel
+	cp ../hail/build/deploy/dist/hail-*-py3-none-any.whl hail/
+	cd hail && tar -cvf wheel-container.tar hail-*-py3-none-any.whl
+	docker build hail -f hail/Dockerfile -t hail-public-image
```

Comment:
do we tag the hail-public-image with the version like 0.2.37?

Reply:
The images built by CI are tagged with the SHA. When I manually upload to DockerHub I do tag with the latest hail version.